site/docs/hosting/firebase.md

```diff
@@ -0,0 +1,43 @@
+# Hosting: Firebase Functions
+
+This tutorial will guide you on how to deploy your bot to [firebase](https://firebase.google.com/) by using [Firebase Serverless Functions](https://firebase.google.com/docs/functions), assuming that you already have a firebase account.
```

Comment:
Don’t assume that the user already has a Firebase account. Guide them on how to create one instead.